tests/pytorch/test_pytorch_autolog.py

```diff
@@ -0,0 +1,158 @@
+import os
+import pytest
+import pytorch_lightning as pl
+import torch
+from iris import IrisClassification
+import mlflow
+import mlflow.pytorch
+from pytorch_lightning.callbacks.early_stopping import EarlyStopping
+from pytorch_lightning.callbacks import ModelCheckpoint
+
+NUM_EPOCHS = 20
+
+
+@pytest.fixture
+def pytorch_model():
+    mlflow.pytorch.autolog()
+    model = IrisClassification()
+    trainer = pl.Trainer(max_epochs=NUM_EPOCHS)
+    trainer.fit(model)
+    client = mlflow.tracking.MlflowClient()
+    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
+    return trainer, run
+
+
+def test_pytorch_autolog_logs_default_params(pytorch_model):
+    _, run = pytorch_model
+    data = run.data
+    assert "learning_rate" in data.params
+    assert "epsilon" in data.params
+    assert "optimizer_name" in data.params
+    assert "weight_decay" in data.params
+    assert "betas" in data.params
+
+
+def test_pytorch_autolog_logs_expected_data(pytorch_model):
+    _, run = pytorch_model
+    data = run.data
+
+    # Checking if metrics are logged
+    assert "loss" in data.metrics
+    assert "val_loss" in data.metrics
+
+    # Testing unwanted parameters are not logged
+    assert "callbacks" not in data.params
```

Comment:
This is a nit, but just because `assert "callbacks" not in data.params` passes doesn't necessarily mean `data.params` doesn't contain unwanted parameters (it might contains a parameter named `"foo"`).